src/pkg/utils/utils_test.go

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2019 – 2021 Red Hat Inc.
```

Comment:
This should just be 2021. The range starts from the year the file (or the code in it) was created.

Reply:
Ah, right.